app/controllers/workshops_controller.rb

```diff
@@ -20,7 +20,7 @@ def show
     @workshop = Workshop.find_by(id: params[:id])
     @user = current_user.presence || User.new
     @registrant = Registrant.new
-    @registrants = find_registrants(@workshop.id)
+    @registrants = Registrant.where(workshop_id: params[:id]).where('registrants.local_school_id = ?', @user.local_school_id)
```

Comment:
What was wrong with the `find_registrants` method? This may have unintended side effects. If `@user` is equal to `User.new` - then `@user.local_school_admin` will be nil, which is a valid search option. There may be `registrants` with a `local_school_id` value of `nil`